src/DashboardHub/Bundle/AppBundle/Resources/config/routing.yml

```diff
@@ -2,6 +2,10 @@ dashboardhub_app_homepage:
     path:     /
     defaults: { _controller: DashboardHubAppBundle:Default:index }
 
+dashboardhub_app_dashboard:
+    path:     /d/{uid}
```

Comment:
True. I did try that, but always had some issue. So parked it all on one route for now.